eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -113,7 +113,10 @@ if ($inputFileToGen) {
 if ($relatedTypeSpecProjectFolder) {
     foreach ($typespecRelativeFolder in $relatedTypeSpecProjectFolder) {
         $typespecFolder = Resolve-Path (Join-Path $swaggerDir $typespecRelativeFolder)
-        $processScript = Resolve-Path (Join-Path "./eng/common/scripts" "TypeSpec-Project-Process.ps1")
+        $processScript = Resolve-Path (Join-Path $sdkPath "eng/common/scripts" "TypeSpec-Project-Process.ps1")
+        $sdkProjectFolders = Get-ChildItem -Path (Join-Path $sdkPath "sdk") -Depth 1 -Directory | Select-Object -ExpandProperty FullName
+
+        # Invoke Process script. SkipSyncAndGenerate only when it's not a new SDK project
         $sdkProjectFolder = & $processScript $typespecFolder $commitid $repoHttpsUrl -SkipSyncAndGenerate
```

Comment:
`SkipSyncAndGenerate` switch would be evaluated along with new sdk project case within `Process` script because we don't want to interact with `tspconfig.yaml` in this script to get `service-dir` and `package-dir`.

Reply:
does this mean that if this flag is there and the `process` script determines its a new project then it will skip sync / gen, but if this flag is not there it will never skip it?

Reply:
Yes, that's correct. sync/gen is skipped when this flag is passed in and it's a new SDK project.